bulk-import/ndjson-generator/NDJsonGenerator.cs

```diff
@@ -0,0 +1,65 @@
+﻿namespace NDJsonGenerator
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Text;
+    using System.Text.Json;
+    using Azure.DigitalTwins.Core.Serialization;
+
+    public class NDJsonGenerator
+    {
+        private const string HeaderSection = "{\"Section\": \"Header\"}";
+        private const string ModelsSection = "{\"Section\": \"Models\"}";
+        private const string TwinsSection = "{\"Section\": \"Twins\"}";
+        private const string RelationshipsSection = "{\"Section\": \"Relationships\"}";
+        private const string DefaultHeader = "{\"fileVersion\": \"1.0.0\", \"author\": \"authorName\", \"organization\": \"organization\"}";
+
+        public static Stream GenerateNDJson(IEnumerable<string> models, IEnumerable<BasicDigitalTwin> twins, IEnumerable<BasicRelationship> relationships, string header = "")
```

Comment:
The name of the function makes it seem as though this is a generic NDJson generator. However, it's very specific to digital twins. Consider renaming the function or parent class.